docassemble/PDFStats/data/static/view_stats.html

```diff
@@ -0,0 +1,281 @@
+    </td>
+    <td>The variation covers 1 standard deviation. See <a href="#flush-collapseThree">the footnotes</a> for more information.</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Consensus reading grade level
+    </th>
+    <td class="{{ get_class("reading grade level") }}">Grade {{ int(stats.get("reading grade level")) | safe }}</td>
+    <td>Target is <a href="https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/style_guide/readability#target-reading-level">4th-6th grade</a><br/>{{ get_data("reading grade level") | safe }}</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Number of pages
+    </th>
+    <td class="{{ get_class("pages") }}">{{ stats.get("pages") | safe }}</td>
+    <td>{{ get_data("pages") | safe }}</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Number of fields
+    </th>
+    <td class="{{ get_class("total fields") | safe }}">{{ stats.get("fields",[]) | length }}</td>
+    <td>{{ get_data("total fields") | safe }}</td>    
+    </tr>
+    <tr>
+    <th scope="row">
+    Average number of fields per page
+    </th>
+    <td class="{{ get_class("avg fields per page") | safe }}">{{ stats.get("avg fields per page",0) | format_number }}</td>
+    <td>Target is < 15<br/>{{ get_data("avg fields per page") }}</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Number of sentences per page
+    </th>
+    <td class="{{ get_class("sentences per page") | safe }}">{{ stats.get("sentences per page") }}</td>
+    <td>{{ get_data("sentences per page") | safe }}</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Word count
+    </th>
+    <td>{{ word_count }} ({{ word_count_per_page | format_number }} per page)</td>
+    <td>Users <a href="https://www.nngroup.com/articles/how-little-do-users-read/">read as little as 20% of the content</a> on a longer page. Try to keep word count to 110 words.</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Number of "difficult words"
+    </th>
+    <td class="{{ get_class("difficult word percent") | safe }}">{{ difficult_word_count }} <br/> ({{ (stats.get("difficult word percent") * 100) | format_number }}%)</td>
+    <td>May include inflections of some "easy" words. Target is < 5% <br/>{{ get_data("difficult word percent") | safe }}</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Percent of sentences with passive voice
+    </th>
+    <td>{{ passive_percent | format_number }}%</td>
+    <td>Target is < 5%</td>
+    </tr>
+    <tr>
+    <th scope="row">
+    Number of citations
+    </th>
+    <td class="{{ get_class("citation count") | safe }}">{{ stats.get("citations",[]) | length }}</td>
+    <td>Avoid using citations in court forms.<br/>{{ get_data("citation count") | safe }}</td>
+    </tr>
+    </tbody>
+</table>
+
+<h2 class="pb-2 border-bottom text-center">Ideas for Improvements</h2>
+{% if stats["suggested title"].lower().strip() != "abortthisnow." %}
```

Comment:
```suggestion
{% if stats.get("suggested title") and stats["suggested title"].lower().strip() != "abortthisnow." %}
```
Could be `None` IIRC